### PR TITLE
Add Terraform v0.14.x support

### PR DESCRIPTION
**Pull request by soerenmartius**
- chore: prepare v0.3.0 release
- test: update terratest to v0.34.0
- build: upgrade build-tools to v0.9.0
- feat: add support for Terraform v0.14.x
- build: update secrets in github actions pipeline
- build: increase timeout for golangci-lint to 5 minutes

### CODE REVIEW

**Review by mariux — APPROVED** [review]
lgtm

**Comment by soerenmartius** [comment]
@mariux bumped  version to `v0.3.0`, increased golangci-lint   timeout to 5min - r4r

**Comment by soerenmartius** [comment]
please review @mariux

**Comment by soerenmartius** [comment]
can we get this merged @mariux ?

**Comment by soerenmartius** [comment]
@mariux needed to temporary downgrade to build-tools v0.8.0 because the docker volumes seemed to be failing since the new GH actions upgrade. Will upgrade to the newest  version in the v0.15 pr

File: README.md [hunk 1]
```diff
@@ -349,7 +349,7 @@ Copyright &copy; 2020 [Mineiros GmbH][homepage]
 [badge-build]: https://github.com/mineiros-io/terraform-aws-ecr/workflows/CI/CD%20Pipeline/badge.svg
 [badge-semver]: https://img.shields.io/github/v/tag/mineiros-io/terraform-aws-ecr.svg?label=latest&sort=semver
 [badge-license]: https://img.shields.io/badge/license-Apache%202.0-brightgreen.svg
-[badge-terraform]: https://img.shields.io/badge/terraform-0.13%20and%200.12.20+-623CE4.svg?logo=terraform
+[badge-terraform]: https://img.shields.io/badge/terraform-0.14%20and%200.12.20+-623CE4.svg?logo=terraform
```
Comment (mariux) [inline README.md]: 0.13 misisng

**Review by mariux — APPROVED** [review]
lgtm

**Pull request merged**